packages/headless/src/controllers/results-per-page/headless-results-per-page.ts

```diff
@@ -61,28 +95,17 @@ export function buildResultsPerPage(
   return {
     ...controller,
 
-    /**
-     * The state of the `ResultsPerPage` controller.
-     */
     get state() {
       return {
-        /** The number of results per page. */
         numberOfResults: engine.state.pagination.numberOfResults,
       };
     },
 
-    /** Updates the number of results to request per page.
-     * @param num The number of results.
-     */
     set(num: number) {
```

Comment:
You can optionally remove the parameter types here and below.

Reply:
True, but I feel it's more consistent to keep the types, and more explicit to see them? If the team prefers to remove them though, we could do a pass and remove them off all controllers in a dedicated PR.